src/python/grpcio/grpc/_cython/_cygrpc/completion_queue.pyx.pxi

```diff
@@ -111,8 +126,8 @@ cdef class CompletionQueue:
         grpc_completion_queue_shutdown(self.c_completion_queue)
       # Pump the queue (All outstanding calls should have been cancelled)
       while not self.is_shutdown:
-        event = grpc_completion_queue_next(
+        event = _grpc_completion_queue_next(
             self.c_completion_queue, c_deadline, NULL)
         self._interpret_event(event)
       grpc_completion_queue_destroy(self.c_completion_queue)
-    grpc_shutdown()
+    grpc_shutdown_blocking()
```

Comment:
Can you please explain the change from an asynchronous shutdown to a synchronous one?

Reply:
I've changed it back, no need for the change.